app/exchanges/run_csv/pacbio_sample_sheet.rb

```diff
@@ -49,9 +49,10 @@ def generate_smrt_cell_settings(well) # rubocop:disable Metrics/MethodLength
         'Indexes'	=> well.barcode_set, # 244d96c6-f3b2-4997-5ae3-23ed33ab925f
         'Sample is indexed'	=> well.tagged?, # Set to True to Multiplex
         'Bio Sample Name' => well.tagged? ? nil : well.bio_sample_name,
-        'Use Adaptive Loading'	=> false, # this will likely be a well default in future
+        'Use Adaptive Loading'	=> well.use_adaptive_loading,
         'Consensus Mode'	=> 'molecule', # (default to molecule do we need a custom field)
-        'Same Barcodes on Both Ends of Sequence'	=> well.same_barcodes_on_both_ends_of_sequence
+        'Same Barcodes on Both Ends of Sequence'	=> well.same_barcodes_on_both_ends_of_sequence,
+        'Full Resolution Base Qual' => well.full_resolution_base_qual
```

Comment:
Just to understand, Why full_resolution_base_qual added to existing version v13?

Reply:
The columns/features have been around since v13 but they are optional but the users have now requested that v13_1 have these fields added too. I guess its because they want to make use of new features.

Reply:
Ohh, ok. I understood, that makes sense. Thank you